setup/windows/README.md

```diff
@@ -49,6 +49,6 @@ $ ansible-playbook -i ../ansible-inventory ansible-playbook.yaml -vv
 ```
 
 The servers should logon automatically at boot and start the Jenkins slave, but there is no automatic restart procedure for crashes (for now).
-Hence, to start Jenkins after running `ansible-playbook`, the server must be restarted or the slave must be manually started with RDP.
```

Comment:
I was under the impression that auto logon never worked if we restart through playbook. 

Also I am a bit confused reading the following line in `README.txt`
```
but there is no automatic restart procedure for crashes (for now).
```

What crashes does this line talk about? Once we add `win_reboot`  step in `common-ansible-playbook.yaml`, I think it will be advisable to just say `"The servers should logon automatically at boot and start the Jenkins slave."` and remove other text/line related to restarting the server? Thoughts?

By the way, I verified adding `win_reboot` step and the auto logon and path resolution works.